### ISSUE TITLE
Turn off device

### ISSUE THREAD

**Issue by Youkier** [dgiese/dustcloud#197]
Hi,

I am looking for a way to turn off remotely the xiaomi robot V1.
Do you know of any raw command to do so ?

Thanks

**Comment by dgiese** [comment]
No, you cant. It is always turned on, as soon as it has power (e.g. docking station)

**Comment by Youkier** [comment]
When it has no power (if the power of the docking station is turned off) it is possible to turn it off by pressing and holding the "ON" button until the vacuum says "turning off".
Do you know if any raw command is doing the same ?

Thanks

**Comment by cryptomilk** [comment]
`ssh root@rockrobo poweroff` ???

**Comment by junior013** [comment]
@cryptomilk It shuts down the linux, but not powers off the device. So, you must power it off manually and restart.
